demos/demos_databases_apis/azure_data_explorer/graphistry_ADX_kusto_demo.ipynb

```diff
@@ -0,0 +1,495 @@
+    "\n",
+    "This new capability is especially powerful for users working with large-scale telemetry, security, and relational datasets natively in Kusto Query Language (KQL).\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## Graphistry Adopts ADX Persistent Graph Semantics\n",
+    "\n",
+    "The graph visualization and investigation platform **Graphistry** has embraced this enhancement in its latest release.\n",
+    "\n",
+    "### What’s New in Graphistry:\n",
+    "- **Native KQL Support**: You can now use KQL directly within Graphistry to run ADX queries and return dataframes.\n",
+    "- **Persistent Graph Queries**: Pass the name of your persistent graph and perform investigations in Graphistry's GPU-accelerated visual interface..\n",
+    "\n",
+    "\n",
+    "### Why It Matters:\n",
+    "With persistent graph semantics and Graphistry’s GPU-accelerated visual interface, analysts can now:\n",
+    "- Query, explore, and visualize ADX graph data with minimal friction.\n",
+    "- Quickly pivot between raw table views and high-context relationship maps.\n",
+    "- Investigate threats, fraud, IoT, and more—faster and more collaboratively.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a761a5b8-5a67-4d18-a72f-f04387aad089",
+   "metadata": {},
+   "source": [
+    "# Getting started\n",
+    "\n",
+    "Getting started with Graphistry is a easy as registering for a free account on [Graphistry Hub](https://hub.graphistry.com) and install Graphistry in your environment.\n",
+    "\n",
+    "Install Graphistry and start your Kusto graph journey.\n",
+    "```bash\n",
+    "pip install graphistry\n",
+    "\n",
+    "```\n",
+    "Python:\n",
+    "```python\n",
+    "import graphistry\n",
+    "\n",
+    "graphistry.register()\n",
+    "graphistry.configure_kusto(cluster=KUSTO_CONF['cluster'], database=KUSTO_CONF['database'])\n",
+    "\n",
+    "dfs = graphistry.kql(\"YourTable | take 5\")\n",
+    "\n",
+    "for df in dfs:\n",
+    "    print(df)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b579c55-a691-49c3-a962-f88982722c00",
+   "metadata": {},
+   "source": [
+    "# Taking it for a spin"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e84ffb6f-baf5-46a4-bb4e-ea0ac6bf0731",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import graphistry"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46524eb4-2781-4996-a8f8-f594f8c2e28d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KUSTO_CONF = {\n",
+    "    \"cluster\": \"https://<clustername>.<region>.kusto.windows.net\",\n",
+    "    \"database\": \"YourDatabase\"\n",
+    "}\n",
+    "\n",
+    "GRAPHISTRY_CONF = {\n",
+    "    \"personal_key_id\": \"YOUR_KEY_ID\",\n",
+    "    \"personal_key_secret\": \"YOUR_SECRET\",\n",
+    "    \"server\": \"hub.graphistry.com\"\n",
+    "}"
```

Comment:
The pattern of using a dict for configs, we can move away from, I reworked Spanner to look like this:
<img width="628" alt="image" src="https://github.com/user-attachments/assets/154b9859-ff30-4b71-8205-22b6b0de7b68" />
<img width="628" alt="image" src="https://github.com/user-attachments/assets/cf1af802-840a-47a2-a6a7-37875e6476f0" />


This has the benefit of IDE code completion w/ typing